From e3e1790c4e8e820654f83e15f5d931c07f1cd4fe Mon Sep 17 00:00:00 2001
From: "chintan.dalwadi" <chintan.dalwadi@gatewaytechnolabs.com>
Date: Mon, 7 Dec 2015 11:11:57 +0530
Subject: [PATCH 1/1] test stag

---
 DemoGitHub/DemoGitHub/Controllers/AccountController.cs | 1 +
 1 file changed, 1 insertion(+)

diff --git a/DemoGitHub/DemoGitHub/Controllers/AccountController.cs b/DemoGitHub/DemoGitHub/Controllers/AccountController.cs
index aeee2e5..08dde9a 100644
--- a/DemoGitHub/DemoGitHub/Controllers/AccountController.cs
+++ b/DemoGitHub/DemoGitHub/Controllers/AccountController.cs
@@ -23,6 +23,7 @@ namespace DemoGitHub.Controllers
         public AccountController(UserManager<ApplicationUser> userManager)
         {
             UserManager = userManager;
+
         }
 
         public UserManager<ApplicationUser> UserManager { get; private set; }
-- 
2.6.3.windows.1

